src/main/mondrian/rolap/SqlConstraintUtils.java

```diff
@@ -757,6 +757,22 @@ public static boolean hasMultiPositionSlicer(List<Member> slicerMembers) {
         return members;
     }
 
```

Comment:
Hey @ivanpogodin, I'd still like to see the above addressed before processing the PR.